crates/stdarch-gen-arm/spec/neon/aarch64.spec.yml

```diff
@@ -2940,11 +2940,9 @@ intrinsics:
       - int8x8_t
       - int8x16_t
     compose:
-      - LLVMLink:
-          name: "rbit.{neon_type}"
-          links:
-            - link: "llvm.bitreverse.{neon_type}"
-              arch: aarch64,arm64ec
+      - FnCall:
+          - simd_bitreverse
+          - - a
 
   - name: "vrbit{neon_type[0].no}"
```

Comment:
it seems possible to use `simd_bitreverse` here too, but I can't get it to generate the right thing.